0_RootFS/Rootfs/bundled/conf/profile.d/bb_utils.sh

```diff
@@ -62,15 +62,40 @@ qfind() {
     find "$@" 2>/dev/null
 }
 
-# Function to install license files
-install_license () {
-    if [ $# -eq 0 ]; then
-        echo "Usage: install_license license_file1.txt [license_file2.md, license_file3.rtf, ...]" >&2
-        exit 1
+# Function to automatically install license files at the end of the build
+auto_install_license () {
+    if [[ -d "${prefix}/share/licenses/${SRC_NAME}" ]]; then
```

Comment:
Eh, my bad, this should be `! -d`.  :P

Reply:
Or `[[ -d ... ]]; then return; fi` if you don't want to tab everything in.

Reply:
My bad too, didn't realise :see_no_evil: Fixed